ml-fundamentals/ml-problem-formulation-envisioning.md

```diff
@@ -0,0 +1,115 @@
+
+The data required to evaluate a potential solution in this case would be which recommendations resulted in new sales, and an estimation of a customer's likeliness to purchase a specific item without a recommendation. Note that, whilst this data could also be used to build a recommendation engine, it is unlikely that this data will be available before a recommendation system has been implemented, so it is likely that we will have to use an alternate data source to build the model.
+
+We can get an initial idea of how to approach a solution to this problem by considering how it would be solved by a subject matter expert. Thinking of how a personal stylist may provide a recommendation, they are likely to recommend items based on one or more of the following:
+
+* generally popular items
+* items similar to those liked/purchased by the customer
+* items that were liked/purchased by similar customers
+* items which are complementary to those owned by the customer
+
+Whilst this list is by no means exhaustive, it provides a good indication of the data that is likely to be useful to us:
+
+* item sales data
+* customer purchase histories
+* customer demographics
+* item descriptions and tags
+* previous outfits, or sets, which have been curated by the stylist
+
+We would then be able to use this data to explore:
+
+* a method of measuring similarity between items
+* a method of measuring similarity between customers
+* a method of measuring how complementary items are relative to one another
+
+which can be used to create and rank recommendations. Depending on the project scope, and available data, one or more of these areas could be selected to create hypotheses to be explored by the data science team. Some examples of such hypothesis statements could be:
+
+* From the descriptions of each item, we can determine a measure of similarity between different items to a degree of accuracy which is specified by a stylist.
+* Based on the behaviour of customers with similar purchasing histories, we are able to predict certain items that a customer is likely to purchase; with a certainty which is greater than random choice.
+* Using sets of items which have previously been sold together, we can formulate rules around the features which determine whether items are complementary or not which can be verified by a stylist.
+
+## References
+
+Many of the ideas presented here - and much more - were inspired by, and can be found in the following resources; all of which are highly recommended.
+
+* Fast.ai's Data project checklist: <https://www.fast.ai/2020/01/07/data-questionnaire/>
```

Comment:
```suggestion
* [Fast.ai's Data project checklist](https://www.fast.ai/2020/01/07/data-questionnaire/)
```

Reply:
The `<` around links were added based on the linter that was recommended in the contribution guidelines, will it complain if I remove these?

Reply:
You got that warning because you had bare urls - but in general we try to use markdown linking - I.e with []() and link the words rather than the urls in the playbook - the suggestion should work